### PR TITLE
completed fcsgx workflow

### PR DESCRIPTION
**Pull request by yumisims**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/sanger-tol/ascc/tree/master/.github/CONTRIBUTING.md)
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: subworkflows/local/run_fcsgx.nf
Comment (weaglesBio) [inline subworkflows/local/run_fcsgx.nf:1]: Could you add the input types, like reference has

**Review by weaglesBio — COMMENTED** [review]
Two changes - including python version and add input details

File: modules/local/parse_fcsgx_result.nf
[the remark this answers is not in the document]
Comment (yumisims) [reply modules/local/parse_fcsgx_result.nf:1]: done

**Review by weaglesBio — APPROVED** [review]
Added changes as suggested

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -2,8 +2,6 @@ name: nf-core CI
 # This workflow runs the pipeline with the minimal test dataset to check that it completes without any syntax errors
 on:
   push:
-    branches:
-      - dev
```
Comment (gq1) [inline .github/workflows/ci.yml:6]: Why this condition being removed? We want the small test to be run for every commit to the dev branch.

File: assets/github_testing/test.yaml [hunk 2]
```diff
@@ -11,8 +11,9 @@ kmer_len: 7
 nt_database: /data/blastdb/Supported/NT/current/
 nt_kraken_db_path: /lustre/scratch123/tol/teams/tola/users/ea10/ascc_databases/nt/nt
 ncbi_taxonomy_path: /lustre/scratch123/tol/teams/tola/users/ea10/databases/taxdump/
-ncbi_rankedlineage_path: /lustre/scratch123/tol/teams/tola/users/ea10/databases/taxdump/rankedlineage.dmp
+ncbi_rankedlineage_path: /home/runner/work/ascc/ascc/ncbi_taxdump/rankedlineage.dmp
```
Comment (gq1) [inline assets/github_testing/test.yaml:14]: if you wan this test to be run on the Github runner, all the /lustre should be replalced.

File: assets/test.yaml [hunk 3]
```diff
@@ -12,6 +12,7 @@ nt_kraken_db_path: /lustre/scratch123/tol/teams/tola/users/ea10/ascc_databases/n
 ncbi_taxonomy_path: /lustre/scratch123/tol/teams/tola/users/ea10/databases/taxdump/
 ncbi_rankedlineage_path: /lustre/scratch123/tol/teams/tola/users/ea10/databases/taxdump/rankedlineage.dmp
 busco_lineages_folder: /lustre/scratch123/tol/resources/busco/data/v5/2021-03-14/lineages
+fcs_gx_database_path: /lustre/scratch124/tol/projects/asg/sub_projects/ncbi_decon/0.4.0/gxdb
 seqkit:
```
Comment (gq1) [inline assets/test.yaml:16]: should this input being used in the full_test profile? It seems not being used currently.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (yumisims) [reply .github/workflows/ci.yml:6]: removed it due to it halts the github pipeline when checking in

**Pull request merged**